apoc/_pixel_classifier.py

```diff
@@ -270,6 +278,29 @@ def _make_features(self, features, image = None):
 
         return features
 
+    def feature_importances(self):
+        """Provide feature importances about the trained Random Forest Classifier
+
+        The values are provided as dictionary {feature_name:portion_importance}.
+
+        See also
+        --------
+        ..[0] https://scikit-learn.org/stable/auto_examples/ensemble/plot_forest_importances.html
+        """
+        opencl_filename = self.opencl_file
+
+        feature_specification = _read_something_from_opencl_file(opencl_filename, "feature_specification =", "") \
+            .replace(",", " ") \
+            .replace("  ", " ") \
+            .strip() \
+            .split(" ")
+
+        result = {}
+        for i, feature in enumerate(feature_specification):
+            result[feature] = self._feature_importances[i]
+
+        return result
```

Comment:
Is it possible for the user to call `feature_importances()` before the values have been populated (e.g., before training)? If so, what happens?

Reply:
Good catch @kevinyamauchi ! That causes an error :-)